workflow/rules/star.smk

```diff
@@ -0,0 +1,35 @@
+__author__ = "Jonas Almlöf"
+__copyright__ = "Copyright 2021, Jonas Almlöf"
+__email__ = "jonas.almlof@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule star:
+    input:
+        fq1=lambda wildcards: alignment_input_fq1(wildcards),
+        fq2=lambda wildcards: alignment_input_fq2(wildcards),
+    output:
+        bam=temp("alignment/star/{sample}_{type}.bam"),
+        sj="alignment/star/{sample}_{type}_SJ.out.tab",
+    params:
+        idx=config.get("reference", {}).get("star_index", ""),
```

Comment:
Maybe we should put this in `input` as well to have snakemake check it before we start running the rule?